### ISSUE TITLE
All text replaced with boxes (NOT USING FAST RENDER)

### ISSUE THREAD

**Issue by Vortex67** [HellFirePvP/AstralSorcery#880]
![image](https://user-images.githubusercontent.com/31183838/46641025-453efe80-cb3d-11e8-84d7-87da2aac7b80.png)
I have seen this issue many times but this time it was not fast render causing it. Any help?

**Comment by Doomgull** [comment]
Does it happen without sponge?

**Comment by Vortex67** [comment]
Yeah, I tried removing sponge, removing optifine and replacing it with foamfix (which crashed my server on launch) and then saw this and tried turning off fast render. Nothing changed

**Comment by Doomgull** [comment]
If you uninstall the pack (back it up first) and reinstall, does it fix it?

**Comment by Vortex67** [comment]
No, the log doesnt seem to show any errors relating to Astral

**Comment by Doomgull** [comment]
Its not a code error it's someone doing weird stuff with rendering, won't be in the logs. Do you recall what started it? May need to use binary method to determine what's triggering it.

**Comment by Vortex67** [comment]
Forgot to mention. Opening the Astral Tome does it but unlike the other issues here its not caused by optifine.
